src/queuing/single_queue_manager.h

```diff
@@ -0,0 +1,193 @@
+#ifndef RECODEX_BROKER_SINGLE_QUEUE_MANAGER_HPP
+#define RECODEX_BROKER_SINGLE_QUEUE_MANAGER_HPP
+
+#include "queue_manager_interface.h"
+
+#include <chrono>
+#include <memory>
+#include <vector>
+#include <map>
+#include <algorithm>
+
+struct request_entry {
+    request_ptr request;
+    std::chrono::milliseconds arrived_at;
+};
+
+struct fcfs_job_comparator {
+    bool compare(const request_entry &a, const request_entry &b, worker_ptr worker) const
+    {
+        return a.arrived_at < b.arrived_at;
+    }
+
+    void drop_caches() {}
```

Comment:
I believe we can safely remove this - it was there to speed up the simulation of another comparator (OAGM, if I recall correctly), which wasn't that great anyways.